generated/mocks/generate.go

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 // mockgen rules for generating mocks for exported interfaces (reflection mode)
-//go:generate sh -c "mockgen -package=fs $PACKAGE/persist/fs FileSetWriter,FileSetReader,FileSetSeeker | mockclean -pkg $PACKAGE/persist/fs -out $GOPATH/src/$PACKAGE/persist/fs/fs_mock.go"
+//go:generate sh -c "mockgen -package=fs $PACKAGE/persist/fs DataFileSetWriter,DataFileSetReader,DataFileSetSeeker,IndexFileSetWriter,IndexFileSetReader,IndexSegmentFileSet,IndexSegmentFile | mockclean -pkg $PACKAGE/persist/fs -out $GOPATH/src/$PACKAGE/persist/fs/fs_mock.go"
```

Comment:
lol fk using `go-generate`, even plain bash over this shit would be an improvement.

Reply:
Yeah true